README.md

```diff
@@ -135,6 +135,53 @@ Parcelgen supports writing and reading the following types to a Parcel:
 }
 ```
 
+### Using Enums
+
+
+Parcelgen provides full support for working with enums. You can add enum fields to your models following the next steps:
+
+**1. Changes required to your .json file**
+
+* create the enum class in any .java file you want and import it
+ 
+``` javascript
+"imports": [
+        "com.example.mypackage.MyEnum"
+    ]
+```
+* use the **json_blacklist** field to let parcelgen know that it shouldn't parse this enum field for you, instead you'll take care of this value and transform the string object contained in your JSON into an instance of your enum.
+``` javascript
+"json_blacklist": [
+        "enumField"
+    ]
+```
+* specify your enum field as being serializable. Parcelgen needs to know this field is serializable so that it knows how to write/read it to/from a parcel object.
+``` javascript
+"serializables": [
+        "MyEnum"
+    ]
+```
+* include your enum into the props list
+``` javascript
+ "props":{
+        "MyEnum": [
+            "enumField"
+        ]
+    }
+```
+
+**2. Changes required to your .java file**
+
+* now it's time to move to your .java generated files. You'll see that parcelgen created 2 Java files for you once you've run the script: ```_Model.java``` and ```Model.java```. Move to ```Model.java``` and override the ```readFromJson()``` method in order to parse the enum field that you've added.
+``` javascript
+    @Override
+    public void readFromJson(JSONObject json) throws JSONException {
+        super.readFromJson(json);
+        mEnumField = MyEnum.getField(json.getString("enum_field"));
+    }
+```
+* provide a method inside your enum class that will convert a string into an instance of your enum. In the example above, this method is called ```getField()```.
```

Comment:
it would be nice, but we cannot cause Java is Camel Case and python not, so conflict here.....(which sucks). It would mean that our enum looks something like:
public enum Enum {
   Test_this_feature; //instead of TestThisFeature
}
It's not really the standard way for java